assets/src/edit-story/components/colorPicker/karma/eyedropper.karma.js

```diff
@@ -52,23 +52,28 @@ describe('Eyedropper', () => {
   it('should get color from the image to page background', async () => {
     // Insert image that will be the color source
     const image = fixture.editor.library.media.item(1);
-    const canvas = fixture.editor.canvas.framesLayer.fullbleed;
-    await fixture.events.mouse.seq(({ down, moveRel, up }) => [
-      moveRel(image, 1, 1),
-      down(),
-      moveRel(canvas, 30, 30),
-      up(),
-    ]);
+    // Only regestering image click by clicking twice for some reason
+    await fixture.events.click(image);
+    await fixture.events.click(image);
 
     // Click the background element
-    await fixture.events.click(
-      fixture.editor.canvas.framesLayer.frames[0].node
+    const bgCanvasRect =
+      fixture.editor.canvas.framesLayer.frames[0].node.getBoundingClientRect();
+    // Make sure to click on part of bg element not covered by media element previously
+    // assed to canvas
+    await fixture.events.mouse.click(
+      bgCanvasRect.left + 10,
+      bgCanvasRect.bottom - 10
```

Comment:
We actually have [`clickOn`](https://github.com/google/web-stories-wp/blob/main/assets/src/karma-fixture/events.js#L314) for that same purpose:

```
await fixture.events.mouse.clickOn(fixture.editor.canvas.framesLayer.frames[0].node, 10 ,10);
```

Reply:
ooo very cool. thanks for sharing!